src/ResourceManager/Network/Commands.Network/help/New-AzureRmVirtualWan.md

```diff
@@ -134,6 +135,13 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -P2SVpnServerConfiguration
+The list of P2SVpnServerConfigurations that are associated with this VirtualWan.
+
+```yaml
```

Comment:
This doesn't have all of the right information... please remove your changes and regenerate.